src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+> paragraph provides a series of concrete instances of `communication`.
+> Crucially, the examples given are all of the kind of communication that occurs
+> between objects; Hegel makes the point to distinguish the kind of
+> communication between objects to something like personality, which “is an
+> infinitely more intense _impenetrability_ [*Harte*] than objects possess”
+> (Hegel 1991, 716). This is an excellent opportunity to look closely at Hegel’s
+> examples as they provide a window into how he thinks about the real-world
+> instances of the logical development. If we spend a bit of time unpacking
+> Hegel’s examples and scrutinising their validity, we will get a clearer
+> understanding of how he thinks of ontology.
+
+Beginning with Hegel’s first example, that of a scent diffusing in an atmosphere
+where there is no restriction on the diffusion of the scent. Why does Hegel
+think that this is a helpful example of how `communication` is exemplified in
+everyday life? At first glance, it makes sense. The way a scent freely diffuses
+itself around an atmosphere so that the whole atmosphere smells of the scent is
+reminiscent of the identical determinateness of the `mechanical object` that
+leads to it becoming a universal. However, how _exactly_ are we supposed to
+understand this? Is the mechanical process at work between the scented molecules
+and the non-scented molecules, or is it just between the scented molecules? To
+pose this question is, I think, to overstep the conceptual restrictions of
+Mechanism and mechanistic thought. The distinction between a scented molecule
+and a non-scented molecule does not obtain within a mechanistic framework
+because, as `mechanical objects`, they do not have different determinations
+(_i.e._ one is scented and the other is non-scented). Quite simply, scented
+molecules and non-scented molecules have an identical determinateness.
+
+### Interlude: How does the Science of Logic relate to External Reality?
+
+A further philosophical conundrum poses itself. Above, I said that the
+distinction between a scented molecule and a non-scented molecule does not
+obtain within a mechanistic framework. This mode of thinking about how the
+logical determinations are reflected in everyday occurrences suggests that the
+same object can be thought of within a mechanistic framework, as well as other
+frameworks. Take the human body, one could conceive of it purely in mechanistic
+terms and theories of its processes in terms of cause and effect, or one could
+conceive of it in organicist terms and conceive of its processes differently.
+Now, what is crucial about this example is that it works because mechanistic
+thinking is more reductive than organicist thinking – the reverse would not be
+possible. One could not conceive of a rock within an organicist framework. Thus,
+the possibility of an object to be conceived within a plurality of frameworks
+requires that the object is open to reductionism.
```

Comment:
Perhaps you could put the point stronger: e.g. "that something is determined as an object means that it must be open to reductionism".

Reply:
That's not quite the point I'm making. I'm saying that an organic object could be reduced such that it is conceived through a mechanistic conception, but a mechanical object could not be "expanded" (just made up that term) such that it is conceived through an organicist conception. Does that make sense?

Reply:
Yep, thanks!